### PR TITLE
Merging changes from master

### PR DESCRIPTION
**Pull request by shaniR**

### CODE REVIEW

File: analyzer/pom.xml [hunk 1]
```diff
@@ -1078,7 +1078,7 @@
     </build>
 
      <properties>
-        <carbon.apim.version>7.0.37-SNAPSHOT</carbon.apim.version>
+        <carbon.apim.version>7.0.39</carbon.apim.version>
```
Comment (harsha89) [inline analyzer/pom.xml]: this should be 7.0.40-SNAPSHOT

Comment (shaniR) [inline analyzer/pom.xml]: should be fixed in master right?

Comment (anuruddhal) [inline analyzer/pom.xml]: shouldn't this be 7.0.40-SNAPSHOT?

**Comment by harsha89** [comment]
@anuruddhal we need to fix this in master

**Pull request merged**